release notes/v0.34.0.md

```diff
@@ -0,0 +1,60 @@
+k6 v0.34.0 is here! :tada: It introduces the already announced `k6/execution` API and includes some more enhancements and a bunch of minor bug fixes.
+
+## New k6 JavaScript API - `k6/execution`
+
+The [k6/execution](https://k6.io/docs/javascript-api/k6-execution) module allows the fetching of information about the current VU, instance (mostly relevant in distributed/cloud) or scenario execution state through a series of exported properties.
+
+### How to use
+
+Some of the previously suggested hacks with the globally available `__VU` and `__ITER` values, such as for [getting a unique object per iteration from an array](https://community.k6.io/t/when-parameterizing-data-how-do-i-not-use-the-same-data-more-than-once-in-a-test/42/2) or [`SharedArray`](https://k6.io/docs/javascript-api/k6-data/sharedarray/), can be done by using the `scenario.iterationInTest` property, which is guaranteed to be unique across VUs, even for distributed or cloud tests. For example:
+
+```
+import exec from "k6/execution";
+import { SharedArray } from "k6/data";
+
+const data = new SharedArray("my dataset", function(){
+	return JSON.parse(open('my-large-dataset.json'));
+})
+
+export const options = {
+    scenarios :{
+        "use-all-the-data": {
+            executor: "shared-iterations",
+            vus: 100,
+            iterations: data.length,
+            maxDuration: "1h"
+        }
+    }
+}
+
+export default function() {
+    // this is unique even in the cloud
+    var item = data[exec.scenario.iterationInTest];
+    http.post("https://httpbin.test.k6.io/amazing-endpoint", item)
+}
+```
+
+**Docs**: [k6/execution](https://k6.io/docs/javascript-api/k6-execution)
+
+## Enhancements and UX improvements
+
+* Warn Windows users on importing dependencies or opening files as absolute paths ([#2078](https://github.com/grafana/k6/pull/2078)).
+* Pass setup data object into [handleSummary callback](https://k6.io/docs/results-visualization/end-of-test-summary/#handlesummary-callback) ([#2103](https://github.com/grafana/k6/pull/2103)). Thanks, @SamuelJohnson01997!
+* Overwrite params using the spread operator ([#824](https://github.com/grafana/k6/issues/824), [#2109](https://github.com/grafana/k6/pull/2109)).
```

Comment:
```suggestion
```

Turns out this wasn't working as expected in the default compatibility mode because of some Babel issues, so we probably shouldn't mention it.